src/controllers/update/update.any.trusts.involved.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { NextFunction, Request, Response } from "express";
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { ApplicationData } from "model";
+import { getApplicationData, setExtraData } from "../../utils/application.data";
+import { AnyTrustsInvolvedKey } from "../../model/data.types.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+    const appData: ApplicationData = getApplicationData(req.session);
+    return res.render(config.UPDATE_ANY_TRUSTS_INVOLVED_PAGE, {
+      backLinkUrl: config.SECURE_UPDATE_FILTER_URL,
+      templateName: config.UPDATE_ANY_TRUSTS_INVOLVED_PAGE,
+      appData,
+      [AnyTrustsInvolvedKey]: appData.any_trusts_involved
+    });
+  } catch (error) {
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+    const appData: ApplicationData = getApplicationData(req.session);
+    const anyTrustsInvolved = req.body[AnyTrustsInvolvedKey];
+    appData.any_trusts_involved = (anyTrustsInvolved) ? + anyTrustsInvolved : 0;
```

Comment:
is this typed as string or number? Maybe "0"? 
And the check:
if (appData.any_trusts_involved === "1")